GithubRepository/src/githubApi.js

```diff
@@ -31,50 +32,46 @@ class GitHubClient {
         }
     }
 
+//Create get Repositories info by organization
 
-    async getTopRepositories(orgName, count = 5) {
-        if (!this.token) {
-            console.error('No se ha proporcionado un token de acceso. Configure la variable de entorno GITHUB_TOKEN.');
-            return [];
+    async getRepositories(orgName , baseUrl){
+        try{
+            const response= await fetch (baseUrl, {
+                header: {
+                    Authorization: 'token ${this.authToken}',
+                },
+            });
+            return await response.json();
+        }catch(error){
+            console.error('Error fetching repositories:', error.message)
+            return null;
         }
+    }
 
-        try {
-            const response = await fetch(`${this.baseUrl}/orgs/${orgName}/repos?per_page=${count}&sort=stars&direction=desc`, {
-                //No traer la data con sorting , solo la data de los repositorios para poner en los mocks 
-                // Y probar el metodo de getTopRepositories
-                headers: {
-                    Authorization: `token ${this.token}`
-                }
-            });
 
-            if (response.ok) {
-                const data = await response.json();
-                return data;
-            } else {
-                console.error('Error al obtener los mejores repositorios:', response.statusText);
-                return [];
-            }
-        } catch (error) {
-            console.error('Error al obtener los mejores repositorios:', error.message);
-            return [];
-        }
+    saveJsonToFile(data, filename) {
+        const jsonData = JSON.stringify(data, null, 2); // Formatear con sangría de 2 espacios
+        fs.writeFileSync(filename, jsonData);
+        console.log(`JSON data saved to ${filename}`);
     }
 
-}
 
 
+}
 
+githubClient = new GitHubClient();
+githubClient.testConnection()
 
+const orgName = 'stackbuilders';
```

Comment:
For constants we use uppercase:
```suggestion
const ORG_NAME = 'stackbuilders';
```